bower.json

```diff
@@ -23,14 +23,18 @@
   ],
   "license": "MIT",
   "ignore": [
+    ".eslintrc",
+    ".gitattributes",
+    ".gitignore",
     ".grunt",
+    ".travis.yml",
     "bower_components",
+    "doc",
+    "GruntFile.js",
+    "layer-selector.gif",
     "node_modules",
-    "tests",
-    ".gitattributes",
-    ".gitignore",
-    ".eslintrc",
-    "GruntFile.js"
+    "README.md",
+    "tests"
```

Comment:
Yes, I also think that we could do a better job of specifying dependencies so that we don't end up with a bunch of resolutions.